### PR TITLE
Add route for initial implementation of coach profile page

### PR DESCRIPTION
**Pull request by ExxZzyy**
Add route for initial implementation of coach profile page

### CODE REVIEW

File: exadel-trainee-app/angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": "76a8f8b5-b301-445d-b4e9-53d44733e940"
```
Comment (shuliakleonid) [inline exadel-trainee-app/angular.json:4]: what is it?

**Review by shuliakleonid — COMMENTED** [review]
see angular.json what is it? i don`t now

File: exadel-trainee-app/angular.json (hunk 1 above)
Comment (ExxZzyy) [reply exadel-trainee-app/angular.json:4]: Without this file the project does not start

Comment (shuliakleonid) [reply exadel-trainee-app/angular.json]: image
Why?

Comment (ExxZzyy) [reply exadel-trainee-app/angular.json]: Fixed

**Review by shuliakleonid — APPROVED** [review]
all clear

**Pull request merged**